ios/KaMPStarteriOS/KoinApplication.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  KoinApplication.swift
+//  KaMPStarteriOS
+//
+//  Created by Russell Wolf on 6/18/20.
+//  Copyright © 2020 Touchlab. All rights reserved.
+//
+
+import Foundation
+import shared
+
+let koinApplication = KoinKt.doInitKoin { it in
```

Comment:
This is still extremely Kotliny and worth further iteration

Reply:
`koinApplication` looks like it can be private. Also I've read that global variables are compute lazily in swift, so I worry that it might be possible for Koin to not get started if you don't reference it in swift